src/gsi/read_l2bufr_mod.f90

```diff
@@ -580,6 +580,7 @@ subroutine radar_bufr_read_all(npe,mype)
 		nobs_badsr=nobs_badsr+1
 		cycle
 	     end if
+             if (.not.radar_box) then
```

Comment:
Please properly indent code from line 583 through line 692.  There is no indentation from line 583 through 602.  The else on line 602 corresponds with if (.not.radar_box) then, so please align with line 583.  Please indent lines 603 through 692 following the realignment of line 602.  The end if  on line 692 corresponds with the else on line 602 and the if statement on line 583.  Please align the end if with the else and if statements.

Reply:
Made changes to the code and pushed changes back to the PR.